addons/editor-dark-mode/addon.json

```diff
@@ -55,6 +600,19 @@
         "value": true
       }
     },
+    {
+      "url": "hide_dots.css",
+      "matches": [
+        "https://scratch.mit.edu/projects/*/",
+        "https://scratch.mit.edu/projects/*/editor",
+        "https://scratch.mit.edu/projects/*/fullscreen",
+        "https://scratch.mit.edu/projects/editor"
```

Comment:
You could simply use `projects` word match

Reply:
I think it's better to do that for all editor addons at once.